.expeditor/verify.pipeline.yml

```diff
@@ -38,4 +38,26 @@ steps:
   expeditor:
     executor:
       docker:
-        image: ruby:3.1
\ No newline at end of file
+        image: ruby:3.1
+
+- label: run-specs-ruby-3.0-windows
+  command:
+    - .expeditor/run_windows_tests.ps1
+  expeditor:
+    executor:
+      docker:
+        host_os: windows
+        shell: ["powershell", "-Command"]
+        image: rubydistros/windows-2019:3.0
+        user: 'NT AUTHORITY\SYSTEM'
```

Comment:
@nikhil2611 from where we can get this user key value

Reply:
This is  Local System account and member of the Windows Administrators group on our  local machine